css/homepage.css

```diff
@@ -0,0 +1,58 @@
+.main-container{
+    margin: 9vh 1rem 0 1rem;
+    display: grid;
+    grid-template-rows: 1fr 3fr 2fr;
+    row-gap: 1rem;
+}
+
+/* .main-container>*{
+    background-color: coral;
+} */
+
+.categories{
+    height:fit-content;
+    width: 100%;
+    display:flex ;
+    flex-direction: row;
```

Comment:
flex direction is by default row so you can remove it if it's not useful.

Reply:
okay, thanks for reviewing.